### DISCUSSION TITLE
Al-ateen Zoom showing as inactive

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1037] [Q&A]
We have a meeting, https://alanonalateen6nc.org/meetings/alateen-wake-forest/?tsml-day=any&tsml-region=wake-forest-nc, that has gone back to Zoom only for now. Since it is Al-ateen they do not publish the zoom link. When I make this change the meeting is showing up as inactive. I saw a similar discussion listing this as a bug but couldn't find a resolution. IS there a way to list this meeting as active without a zoom link?

Thank you
Rose L

**Comment by joshreisner** [comment]
Hi Rose, by default a meeting won't be considered online if there is no online info present.

Is there a password on the meeting? Although we don't recommend it, one simple workaround might be to publish the meeting link without the password, eg `https://zoom.us/j/0192830123` without the `?pwd=o8saruyiw37hfls83qhk`

Another, more technical workaround might be to publish a new 'support' page on your website that instructs users to contact district6altc@gmail.com to request an invitation to an alateen meeting, and then add that page's full URL as the conference URL. To make this work, you will need to add the following to your theme's functions.php:

```
$tsml_conference_providers = array_merge($tsml_conference_providers, [
    'alanonalateen6nc.org' => 'Invitation Only'
]);
```

Then the button will say "Join with Invitation Only" and link to the page you just created.

**Comment by district6wc** [comment]
Hi

I added this code to the functions file in themes and then linking to the
online meeting section, I am not getting an Invitation only button? I did
add the link to Website input also and that just created a button with the
link text. This is the entry I am working on
https://alanonalateen6nc.org/meetings/alateen-wake-forest/. Hoping you can
clarify how to get that button update
[image: image.png]

[image: image.png]



Thank you.
Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
sorry to hear! i tried it just now on our demo site and it seems to work well, here are the steps i took:

1. added the code to our theme's functions.php (note this is a teensy bit dangerous, best if you also have FTP or File Manager access to this file in case something goes wrong). i added an if statement surrounding it, you might want to copy this code and modify for your domain. i'll paste it at the bottom
![1-edit-theme](https://user-images.githubusercontent.com/1551689/212391599-ce692e5f-a965-4236-b210-111b53b2ac96.png)

2. enter conference_url (note which field, make sure the domain matches exactly)
![2-enter-url](https://user-images.githubusercontent.com/1551689/212391851-fd61fc63-ed19-45bd-9593-312fb955854d.png)

3 click view meeting
![3-view-meeting](https://user-images.githubusercontent.com/1551689/212391887-ad9a07cf-447a-46f1-947e-56c18089e178.png)


here's the full code i recommend. also double check that the URL you're entering is exactly `https://alanonalateen6nc.org/alateen-coordinator-contact/` (or whatever the page URL is)

```
if (isset($tsml_conference_providers)) {
	$tsml_conference_providers = array_merge($tsml_conference_providers, [
		'alanonalateen6nc.org' => 'Invitation Only'
	]);
}
```

**Comment by district6wc** [comment]
Hi  Thank you! I think I was not closing the previous string, this is
working now.
I always appreciate your quick and clear support.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Comment by OSAwebsite** [comment]
Hi,

I am trying to do this for an Alateen meeting that we have on our website. When I enter the substitute page in the Zoom URL, the plugin is not saving it. Any suggestions?

<img width="940" height="362" alt="Screenshot 2025-12-02 at 8 14 20 PM" src="https://github.com/user-attachments/assets/4e2c02c5-7e7e-42c8-a506-5b46dc9cf7b2" />

The group is very anxious to have their meeting show as "Active" on the site

**Reply by SIA-WEBSUP** [reply]
@OSAwebsite, can you give us the **exact URL** you entered?
Did you add that domain to the `$tsml_conference_providers` array as described above?

And if the plugin is running on a different domain, would you give us the URL to your `/meetings` page?

**Reply by SIA-WEBSUP** [reply]
Hey @OSAwebsite I just added a meeting to our demo site following the instructions above and things worked as expect.
- https://demo.code4recovery.org/meetings/alateen-invitation-only?search=alateen

Make sure you are providing an 'approximate' address if it's an online meeting.

**Comment by OSAwebsite** [comment]
Hi,

The domain for our meetings listing is https://al-anon.alateen.on.ca/meetings/

The url I attempted to insert in the meeting listing is - https://al-anon.alateen.on.ca/invitation-only-meetings/

The meeting listing does have an approximate address listed :) It is the same one we use for all our PEGs

**Reply by OSAwebsite** [reply]
I also added the domain to the array as described above. We use an Astra template for our website

**Reply by SIA-WEBSUP** [reply]
I was able to update my test meeting on our demo site with your domain with no issues.
- see: https://demo.code4recovery.org/meetings/alateen-invitation-only

Here's what the meeting information looks like on my test meeting:
<img width="1290" height="988" alt="image" src="https://github.com/user-attachments/assets/6164af64-3941-4931-9e60-1f3bcbf2b5ee" />


Here's what the override looks like in my theme's functions.php:
```

if (isset($tsml_conference_providers)) {
	$tsml_conference_providers = array_merge($tsml_conference_providers, [
		'al-anon.alateen.on.ca' => 'Invitation Only -  Eh!'
	]);
}
```

Does yours look similar?

**Reply by SIA-WEBSUP** [reply]
If everything looks similar to what's above... then perhaps it is some weird interaction with the Astra theme that prevents the code from running when it should.

Do you have any other TSML related tweaks in your functions.php? Do they work as expected?

**Reply by SIA-WEBSUP** [reply]
@OSAwebsite, just following up on your issue. I noticed that [Hope Prevails Alateen - Mississauga (D16)](https://al-anon.alateen.on.ca/meetings/?search=hope+prevails) is still listed as inactive. Have you had a chance to check the code snippet that adds your domain to the list of permitted [video] conference providers? Is it pretty much identical to what I used on our test site?
